pkg/standalone-utils/contracts/utils/HyperSpotPricePrecompile.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+library HyperSpotPricePrecompile {
+    address public constant SPOT_PRICE_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000808; // spotPx
```

Comment:
Should have some space between these, and a comment on the error. Maybe also a block comment at the top saying where this comes from / how it's used (e.g., https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/hyperevm/interacting-with-hypercore).

Same below.